libsolidity/parsing/DocStringParser.h

```diff
@@ -35,7 +35,7 @@ class ErrorReporter;
 namespace solidity::frontend
 {
 
-class StructurallyDocumented;
+class StructuredDocumentation;
```

Comment:
Is `StructuredDocumentation` even needed there? Looks like it worked just fine without it so far.

Reply:
I think because the header was only included in a place where it was already defined, but if we want this header to be valid standalone, this is needed.